AMPDemos/index.html

```diff
@@ -5,11 +5,10 @@
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1,user-scalable=no" />
   <title>AMP Demos</title>
-  <script async src="https://www.googletagmanager.com/gtag/js?id=UA-114121901-1"></script>
-  <script src="idb.js"></script>
-  <script>
+<script async src="https://www.googletagmanager.com/gtag/js?id=UA-114121901-1"></script>
+  <script src="/idb.js"></script>
```

Comment:
this needs to be relative (i.e. `idb.js`) since it's served on https://mdiblasio.github.io/AMPDemos/ and file is hosted at https://mdiblasio.github.io/AMPDemos/idb.js

Reply:
@noodles

Reply:
Sorry - I was being lazy and testing this on Glitch. Forgot to change it back.